data/data/agent/systemd/units/iscsistart.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Initiates iSCSI Boot Process by Preparing and Connecting to iSCSI Target
```

Comment:
should this service be only started based on a conditionPathExists or by checking an env var which could signal ABI is running on OCI?

Reply:
I assume we want to allow iSCSI even when we are not on OCI.

Reply:
Yes, it's not just OCI the intention is to add iSCSI support to the platforms that Agent can deploy.

Reply:
Thank you for the clarification.